### PR TITLE
Fix: transfer asset does't successfully but for using default account

### PR DESCRIPTION
**Pull request by Yihen-Liu**
Signed-off-by: liuyihen@onchain.com

### CODE REVIEW

File: cmd/asset_cmd.go [hunk 1]
```diff
@@ -187,7 +187,12 @@ func transferAsset(ctx *cli.Context) error {
 	}
 
 	acct := account.Open(account.WALLET_FILENAME, passwd)
```
Comment (laizy) [inline cmd/asset_cmd.go]: need check `acct == nil` to avoid panic

**Review by laizy — CHANGES_REQUESTED** [review]

File: cmd/asset_cmd.go [hunk 2]
```diff
@@ -203,17 +203,21 @@ func transferAsset(ctx *cli.Context) error {
...
+		fmt.Println("Open account failed, please check your input password and make sure your wallet.dat exist")
+		return errors.New("Get Account Error")
 	}
-	acct := account.Open(filename, passwd)
-	if acct != nil {
-		return errors.New("open wallet error")
+
+	addr, err := common.AddressFromBase58(from)
+	if nil != err {
+		fmt.Println("Parase address from base58 error")
```
Comment (laizy) [inline cmd/asset_cmd.go]: `:s/Parase/Parse`

**Review by laizy — CHANGES_REQUESTED** [review]

File: cmd/asset_cmd.go (hunk 2 above)
Comment (Yihen-Liu) [reply cmd/asset_cmd.go]: what a shame. Have modified.

**Review by Arbio5zt — APPROVED** [review]
approve

**Pull request merged**